### PR TITLE
[MOB-3044] FloatingPanel 2.0.1 fix

### PR DESCRIPTION
**Pull request by EdwardW2**
## JIRA
[MOB-3044](https://karhoo.atlassian.net/browse/MOB-3044)

## DESCRIPTION
Fix for FloatingPanel 2.0.1 pod

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [ ] New feature 
- [ ] Tech Debt
- [x] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [x] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [x] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

**Review by nurito-burrito — APPROVED** [review]

File: KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteListPanelLayout.swift [hunk 1]
```diff
@@ -9,27 +9,23 @@
...
-    func insetFor(position: FloatingPanelPosition) -> CGFloat? {
-        switch position {
-        case .full: return QuoteListPanelLayout.topOffset
-        case .half: return QuoteListPanelLayout.compactSize
-        default: return nil
-        }
-    }
-
-    func backdropAlphaFor(position: FloatingPanelPosition) -> CGFloat {
+    //TODO: backdropAlphaFor is now backdropAlpha(for: FloatingPanelState) -> CGFloat which does not work with the current switch statement
```
Comment (JeevanThandi) [inline KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteListPanelLayout.swift:27]: is that fixed now? seems like it should work

File: KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift [hunk 2]
```diff
@@ -125,9 +125,21 @@ final class KarhooBookingViewController: UIViewController, BookingView {
 
     private func setupQuoteListPanel() {
         let mainPanelVC = FloatingPanelController()
+        
+        let appearance = SurfaceAppearance()
```
Comment (JeevanThandi) [inline KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift:129]: does this massively change appearance ? able to attach a screenshot to the PR if so?

File: KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteListPanelLayout.swift (hunk 1 above)
Comment (EdwardW2) [reply KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteListPanelLayout.swift:27]: Yeah, just left the comment there for reference in the future if it changes again according to the Migration guide from v1 to v2

File: KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift (hunk 2 above)
Comment (EdwardW2) [reply KarhooUISDK/Screens/BookingScreen/KarhooBookingViewController.swift:129]: Nope, didn't change the appearance, this is mainly for the shadow. The Quote view still looks like it did before.

**Pull request merged**